lib/loggerator/railtie/log_subscriber.rb

```diff
@@ -0,0 +1,85 @@
+# Implementation respectfully borrowed from:
```

Comment:
I want to push this PR forward, however, I created an issue on it... https://github.com/heroku/loggerator/issues/5